components/org.wso2.carbon.siddhi.metrics.core/src/main/java/org/wso2/carbon/siddhi/metrics/core/internal/SiddhiMetricsManagement.java

```diff
@@ -0,0 +1,68 @@
+package org.wso2.carbon.siddhi.metrics.core.internal;
```

Comment:
Add license header.

Reply:
Fixed in bbc4d5bc48eab76b230605ce1d56e75b4e558ffb